colossalai/kernel/cuda_native/csrc/smoothquant/linear.cu

```diff
@@ -0,0 +1,162 @@
+// modified from https://github.com/Guangxuan-Xiao/torch-int/blob/main/torch_int/kernels/linear.cu
```

Comment:
Can you import this lib directly?

Reply:
The linear in torch int lib only support relu linear.  This linear file is to support silu linear.

Reply:
Then shall we add their license to ours? And do they allow modification?

Reply:
They use MIT License. I add the license for torch-in and smoothquant in the LICENSE